openfisca_survey_manager/__init__.py

```diff
@@ -43,7 +43,7 @@
 with open(os.path.join(test_config_files_directory, 'config.ini'), "w+") as file:
     file.write(config_ini)
 
-# GitHub actions test
+# GitHub Actions test
 if 'CI' in os.environ:
```

Comment:
En une ligne :
```suggestion
 is_in_ci = 'CI' in os.environ
```

Reply:
Done in 430b7875394cadbf842526e2798be43b6f839e5a (to update the `else` line under the suggestion).